examples/docker-compose.yml

```diff
@@ -2,13 +2,20 @@ version: '3.4'
 
 services:
   bds:
-    image: itzg/minecraft-bedrock-server
+    image: foleymic/minecraft-bedrock-server
```

Comment:
Need to revert this :)

Reply:
Opps.  Long day and was rushing a bit.  Forgot to revert.